std_msgs/Header header                  # time stamp
int16[3] acc        # Accelerometer reading -  Acceleration in m/sec^2